packages/cozy-harvest-lib/src/components/TriggerLauncher.jsx

```diff
@@ -112,6 +116,11 @@ export class TriggerLauncher extends Component {
     this.displayTwoFAModal()
   }
 
+  async handleTwoFACode(code) {
+    this.setState({ isSubmittingTwoFA: true })
```

Comment:
It seems to me that we are duplicating the submittingTwoFa state, at some place we use isTwoFaRunning() and at another place we use `isSubmittingTwoFa` here ?
BTW is submitting and running the same thing ? If it is, let's use the same word, it will be clearer.

Reply:
It seeems `isSubmittingTwoFA` is not used here and only used to trigger a re-render.

Maybe we need an observable on konnectorJob ? `konnectorJob.on(change, this.forceUpdate)` ?